cypress/e2e/cloud/geoOptions.test.ts

```diff
@@ -9,6 +9,11 @@ describe('DataExplorer - Geo Map Type Customization Options', () => {
         cy.fixture('routes').then(({orgs, explorer}) => {
           cy.visit(`${orgs}/${id}${explorer}`)
           cy.getByTestID('tree-nav')
+          cy.window().then(w => {
+            // I hate to add this, but the influx object isn't ready yet
```

Comment:
this is crazy. it's loaded like.. first. you'd think the visit would be good enough, but i guess if it's only used behind the first setorg suspense and yadda yadda...

Reply:
yeah it's super annoying. I tried waiting for other elements on the page too but nothing would do the trick. I figured these flags will be removed soon enough and then we can delete this code.